packages/messaging/README.md

```diff
@@ -34,16 +34,17 @@ npm install {path to node package file}
 ```
 
 **Podfile setup for `@adobe/react-native-aepmessaging` v1.0.0-beta.2 with in-app messaging support**  
-The In app Message APIs depends on the AEP Messaging, v1.1.0. This version is not yet published to the Cocoapods but is available in the public [github repository](https://github.com/adobe/aepsdk-messaging-ios/tree/staging). Add the following pod dependency in your applications Podfile under the application target. 
+The In app Message APIs depends on the AEP Messaging, v1.1.0. This version is not yet published to the Cocoapods but is available in the public [github repository](https://github.com/adobe/aepsdk-messaging-ios/tree/staging). Add the following pod dependency in your applications Podfile under the application target.
 
 ```ruby
 target 'AEPSampleApp' do
 pod "AEPMessaging", :git => "https://github.com/adobe/aepsdk-messaging-ios.git", :branch => "staging"
+pod "AEPOptimize", :git => "https://github.com/adobe/aepsdk-optimize-ios.git", :branch => "staging"
 end
 ```
 
 **Gradle setup for `@adobe/react-native-aepmessaging` v1.0.0-beta.2 with in-app messaging support**
-AEPMessaging Android package v1.1.0 with in-app messaging support is not published to maven yet. In project level build.gradle file of Android project in your RN application add following changes in `allprojects -> repositories`. 
+AEPMessaging Android package v1.1.0 with in-app messaging support is not published to maven yet. In project level build.gradle file of Android project in your RN application add following changes in `allprojects -> repositories`.
```

Comment:
1. AEPMessaging Android package v1.1.0 with in-app messaging support is published to maven snapshots. In project level build.gradle file of Android project in your RN application add  `maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }`   in `allprojects -> repositories`.